src/gfx/fbxExport.js

```diff
@@ -307,30 +307,36 @@ function _writeToFile(filename, data, isBinary, append, callback) {
   }, _errorHandler);
 }
 
-function Queue() { }
-
-Queue.prototype.queue = [];
-Queue.prototype.busy = false;
+/* Temporary note: previously this variables were defined in non-standard way:
```

Comment:
Hey! Thank you for the changes.
I leave a few comments here.
At first, please, remove the comments.